lib/gems/pending/util/miq_apache/miq_apache.rb

```diff
@@ -266,15 +266,21 @@ def add_ports(ports, protocol)
 
       ports = Array(ports).sort.reverse
       ports.each do |port|
-        @raw_lines.insert(index + 1, "BalancerMember #{protocol}://0.0.0.0:#{port}\n")
+        # Temporarily disable the connection reuse for WebSockets for httpd version < 2.4.25
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1404354
+        if protocol == 'ws'
+          @raw_lines.insert(index + 1, "BalancerMember #{protocol}://0.0.0.0:#{port} disablereuse=on\n")
+        else
+          @raw_lines.insert(index + 1, "BalancerMember #{protocol}://0.0.0.0:#{port}\n")
+        end
       end
       ports
     end
 
     def remove_ports(ports, protocol)
       ports = Array(ports)
       ports.each do |port|
-        @raw_lines.delete_if { |line| line =~ /BalancerMember\s+#{protocol}:\/\/0\.0\.0\.0:#{port}$/ }
+        @raw_lines.delete_if { |line| line =~ /BalancerMember\s+#{protocol}:\/\/0\.0\.0\.0:#{port}( disablereuse=on)?$/ }
```

Comment:
@skateman does this mean we only remove the `ws` protocol BalancerMembers?  What about non-ws?  I guess I'm wondering why we are checking for disablereuse at all if no other line will match the protocol, 0.0.0.0 and port... Right?